test/iconGenerator/functionality.ts

```diff
@@ -0,0 +1,174 @@
+/* tslint:disable only-arrow-functions */
```

Comment:
is there a reason why this file has not a `test.ts` extension? I already merged it but I can change it in the pending 7.3.0 PR.